diff --git a/plat/renesas/rz/soc/g3s/include/rz_soc_def.h b/plat/renesas/rz/soc/g3s/include/rz_soc_def.h
index 4b6c797..a919e6e 100644
--- a/plat/renesas/rz/soc/g3s/include/rz_soc_def.h
+++ b/plat/renesas/rz/soc/g3s/include/rz_soc_def.h
@@ -61,7 +61,7 @@
 #define RZG3S_ASRAM_SIZE				UL(0x00080000)
 #define RZG3S_ASRAM_LIMIT				(RZG3S_ASRAM_0_BASE + RZG3S_ASRAM_SIZE)
 #define RZG3S_SRAM_BASE					(RZG3S_MSRAM_0_BASE)
-#define RZG3S_SRAM_SIZE					(RZG3S_MSRAM_SIZE + RZG3S_ASRAM_SIZE)
+#define RZG3S_SRAM_SIZE					(RZG3S_MSRAM_SIZE + RZG3S_ASRAM_SIZE + 0x2000)
 #define RZG3S_DDR0_SIZE					ULL(0x40000000)
 
 #define RZG3S_NS_DRAM_BASE				ULL(0x48000000)
